src/shared/flag-details/rules.tsx

```diff
@@ -1,121 +1,129 @@
+                variance={variance}
+                index={field.key}
+                key={field.key}
+                remove={() => remove(field.name)}
+              />
+            ))}
 
-      <Col>Default Rule</Col>
-      <Col span={24}>
-        <Card>
-          <VarianceSelect
-            itemProps={{
-              name: "fallthrough.variation",
-              initialValue: fallthroughVariance,
-            }}
-            variance={variance}
-            disabled={false}
-          />
-        </Card>
-      </Col>
-    </Row>
+            <Form.Item>
+              <Button
+                block
+                type="dashed"
+                onClick={() =>
+                  add({
+                    variation: 0,
+                    trackEvents: false,
+                    clauses: [
+                      {
+                        attribute: "email",
+                        op: "endsWith",
+                        values: [],
+                        negate: false,
+                      },
+                    ],
+                  })
+                }
+                icon={<PlusOutlined />}
+              >
+                Add sights
+              </Button>
+            </Form.Item>
+          </Space>
+        )}
+      </Form.List>
+
+      <Card>
+        <VarianceSelect
+          itemProps={{
+            name: "fallthrough.variation",
+            label: "Default Rule",
+          }}
+          variance={variance}
+          disabled={false}
+        />
+      </Card>
+    </>
   );
 }
 
 type RuleProps = {
-  rule: FlagDetails_flagDetails_rules;
   variance: boolean[];
   index: number;
+  remove: () => void;
 };
 
-const Rule: React.FC<RuleProps> = ({ rule, variance, index }) => {
-  const clause = rule.clauses[0];
+const Rule: React.FC<RuleProps> = ({ variance, index, remove }) => {
   return (
     <Card>
-      <Form.Item name={`rules.${index}.id`} initialValue={rule.id} hidden>
+      <Row gutter={[4, 20]}>
+        <Col flex="auto">
+          <Typography.Title level={5}>Rule {index + 1}</Typography.Title>
+        </Col>
+        <Col flex="32px">
+          <Dropdown
+            arrow
+            placement="bottomRight"
+            overlay={
+              <Menu onClick={() => remove()}>
+                <Menu.Item>Delete rule</Menu.Item>
+              </Menu>
+            }
+          >
+            <Button type="text" shape="circle" icon={<MoreOutlined />} />
+          </Dropdown>
+        </Col>
+      </Row>
+
+      <Form.Item name={["rules", index, "id"]} hidden>
         <Input />
       </Form.Item>
 
-      <Row gutter={4}>
+      <Row gutter={[4, 20]} align="middle">
         <Col span={1}>IF</Col>
 
         <Col span={3}>
-          <Form.Item
-            name={`rules.${index}.clauses.0.attribute`}
-            initialValue={clause.attribute}
-          >
+          <Form.Item noStyle name={[index, "clauses", 0, "attribute"]}>
             <Select disabled style={{ width: "100%" }} />
```

Comment:
why Select is disabled ?

Reply:
i dont know it ,   maybe we don't have option for it  ? @puncsky

Reply:
because for now we don't support other kinds of attribute so the only choice is hard coded and disabled 

we could re-enable them when more items are added in the future